### DISCUSSION TITLE
[Suggestion] Delete all sourcecode

### DISCUSSION THREAD

**Discussion by Motophan** [darthcloud/BlueRetro#902] [General]
Users are able to bypass the paywall by just compiling the firmware bin file. 

Problem: users are not required to pay for the software if they are able to compile 
Solution: remove all source files :)

w/ zero expirence w/ firmware I was able to compile 1.9.1 in under 5 minutes and flash my adapter using the web OTA. 
![MLGnqz5](https://github.com/darthcloud/BlueRetro/assets/42502676/0ad56d5c-bead-4884-83d1-4f635105356f)

Alternatively allow donations w/ a privacy cryptocurrency coin so users do not have to dox themselves to use your software.

**Comment by darthcloud** [comment]
That's not a problem, its a voluntary contribution that can be bypassed directly on itch.io.

![Screenshot_20240201-232049](https://github.com/darthcloud/BlueRetro/assets/3744056/dbbf56c6-1970-4fee-b3b2-3896ab3999cd)

**Comment by Motophan** [comment]
I misunderstood how itch.io worked due to my poor english skills

Do you have a xmr address? I cannot use traditional payment methods. 
Can you add crypto public address at bottom of github page?
